src/chrome/content/rules/Deutsche-Rentenversicherung.de.xml

```diff
@@ -0,0 +1,26 @@
+<ruleset name="Deutsche-Rentenversicherung.de">
+
+	<target host=       "deutsche-rentenversicherung.de" />
+	<target host=   "www.deutsche-rentenversicherung.de" />
+	<target host="static.deutsche-rentenversicherung-bund.de" />
+	<target host=   "www.deutsche-rentenversicherung-bund.de" />
+	<target host=       "deutsche-rentenversicherung-bund.de" />
+	<target host="www.drv-bund.de" />
+	<target host=    "drv-bund.de" />
+
+  	<securecookie host="^www\.deutsche-rentenversicherung\.de$" name=".+" />
+
+	<!-- could not resolve host -->
+	<rule from="^http://deutsche-rentenversicherung(-bund)?\.de/"
```

Comment:
When rewriting to add the www, it's a little nicer to keep the rest of the URL the same when possible, since it's easier to trivially validate that the redirect is correct.